### ISSUE TITLE
Refactor the cli.py file

### ISSUE THREAD

**Issue by Artmorse** [Schrodinger-Hat/ImageGoNord-cli#19]
The code should be refactored a lot.

We can also use [argparse](https://docs.python.org/3/library/argparse.html) to make argument management easier.

**Comment by Wabri** [comment]
I think @BugliL have done something or maybe have thought about the refactoring using argparse, do you remember if you have done it or am I crazy @BugliL?

**Comment by BugliL** [comment]
Yes I am, but I am a bit stuck, I don't have the headspace to seriously focus on this.  
I opened this PR https://github.com/BugliL/ImageGoNord-cli/pull/3 to show you my work. I will try to finish it and open a PR shortly for everyone.

**Comment by Wabri** [comment]
Huge work, thanks! We'll wait for it :guitar:
